types/types.go

```diff
@@ -101,10 +101,10 @@ type Package struct {
 	// 'package x' line.
 	Name string
 
-	// DocComments from doc.go, if any.
+	// The comment right above the package declaration in doc.go, if any.
 	DocComments []string
 
-	// Comments from doc.go, if any.
+	// All comments from doc.go, if any.
```

Comment:
Sure. I was trying to interpret what's the original intention. I can remove one of them.